tests/retryable-reads/retryable-reads-001.phpt

```diff
@@ -0,0 +1,58 @@
+--TEST--
+Retryable reads: supported single-statement operations include transaction IDs
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongoc_crypto(); ?>
+<?php skip_if_server_version('<', '3.6'); ?>
```

Comment:
I think this can be removed, as `skip_if_no_failcommand_failpoint()` already checks the server version. If the URI is mongos, we require 4.1.8+; otherwise, 4.0+.